go-controller/pkg/libovsdbops/transact.go

```diff
@@ -38,7 +38,7 @@ func TransactAndCheck(c client.Client, ops []ovsdb.Operation) ([]ovsdb.Operation
 		return []ovsdb.OperationResult{{}}, nil
 	}
 
-	klog.Infof("Configuring OVN: %+v", ops)
+	klog.V(4).Infof("Configuring OVN: %+v", ops)
```

Comment:
on second thoughts, can we make this V(5) ? Because it seems we already have :
```
	o.logger.V(4).Info("transacting operations", "database", dbName, "operations", fmt.Sprintf("%+v", operation))

```
So configuring OVN is not needed at level 4, 

- this was discussed with the wider team and all agree.

Reply:
alright.
The back story is we have a large number of address_set and our logs get flooded by these 2 logs. While reproducing in local dev, pprof was showing klog as a top consumer.
However upon further investigation, it appears there is a different issue - if the namespace was deleted during the processing of netpols, OVN Kubernetes seem to retry forever the transactions. So this explains the flood. Reducing the verbosity or amount of logs in itself is less interesting that what I initially thought...

Reply:
this should be fixed by https://github.com/ovn-org/ovn-kubernetes/pull/3161 (still on review)

Reply:
@freedge can you share some logs to check that this case is handled properly now?